### ISSUE TITLE
Will you release the training code of VAE.

### ISSUE THREAD

**Issue by serend1p1ty** [THUDM/CogVideo#186]
### Feature request  / 功能建议

There is not much description of the training details of VAE in the paper, including the weights of various losses, the implementation of discriminators, learning rate scheduler, and so on.

Will you release the training code of VAE?

### Motivation / 动机

Reproduce your VAE.

### Your contribution / 您的贡献

None

**Comment by zRzRzRzRzRzRzR** [comment]
We currently don't have any related plans, but we are working on fine-tuning code based on Diffusers models. We can consider adding fine-tuning for VAE as well.

**Comment by serend1p1ty** [comment]
Thanks for your reply.

**Comment by serend1p1ty** [comment]
I have train a stronger VAE than CogVideoX using Open-Sora-Plan's code.
